sql/wsrep_dummy.cc

```diff
@@ -125,6 +125,12 @@ longlong wsrep_thd_trx_seqno(THD *)
 struct wsrep_ws_handle* wsrep_thd_ws_handle(THD *)
 { return 0; }
 
+void wsrep_thd_set_split(THD *thd, bool split)
+{ }
+
+bool wsrep_thd_get_split(THD *thd)
+{ return 0; }
+
 int wsrep_trx_is_aborting(THD *)
 { return 0; }
```

Comment:
I would pair `bool` with `false`, not `0`. But, is there a reason to use `int` for Boolean values? The name `wsrep_trx_is_aborting()` suggests that it is a Boolean predicate, yet it is returning `int`.

I don’t think that `thd_` needs to be part of the function names.
I’d prefer more descriptive names, such as `wsrep_is_load_multi_commit()` and `wsrep_set_load_multi_commit()`.